runtime/webhookserver/config.go

```diff
@@ -107,6 +119,15 @@ func NewServer(config interface{}) (Server, error) {
 	if schematypes.MustMap(localtunnelConfigSchema, config, &c) == nil {
 		return NewLocalTunnel(c.BaseURL)
 	}
+	if schematypes.MustMap(webhooktunnelConfigSchema, config, &c) == nil {
+		authCl := auth.New(credentials)
+		whresp, err := authCl.WebhooktunnelToken()
+		if err != nil {
+			return nil, err
+		}
+		s, err := NewWebhookTunnel(whresp)
```

Comment:
Do we need to refresh a token? It could be useful when proxy secrets are rotated, but the webhooktunnel is only instantiated once when every worker starts ( https://github.com/taskcluster/taskcluster-worker/blob/master/worker/worker.go#L84 ), so I'm not quite sure how we would reconnect. A solution could be to pass this into the authorizer, but that would require refactoring of whclient.